README.md

```diff
@@ -1,7 +1,15 @@
-# Community-events
+# Community Events @ 🤗
 
-This repository is meant for all community events organized by 🤗 HuggingFace.
+A central repository for all community events organized by 🤗 HuggingFace. Come one, come all!
+We're constantly finding ways to democratise the use of ML across modalities and languages. This repo contains information about all past, present and upcoming events.
 
-## Content
+## Hugging Events
 
-May 16 - May 31: [Blocks Party 🎊](gradio-blocks)
+| **Event Name**                                                          | **Dates**       | **Status**                                                                                                   |
+|-------------------------------------------------------------------------|-----------------|--------------------------------------------------------------------------------------------------------------|
+| [Whisper Fine Tuning Sprint](/whisper-fine-tuning-sprint)               | Dec 5th - 19th, 2022  | [Registrations open](https://discuss.huggingface.co/t/open-to-the-community-whisper-fine-tuning-event/26681) |
```

Comment:
(renamed this to "Whisper Fine Tuning Event" and "/whisper-fine-tuning-event")

Reply:
See:
https://github.com/huggingface/community-events/pull/39/commits/e3f0554373bd272543cb21ce1d789fb93c650212

Reply:
Fixed! Thanks for catching that! 🤗